content/en/blog/2024/demo-skyscanner/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title:
+  'Making observability fun: How we increased engineers confidence in incident
+  management using a game'
```

Comment:
Looking at how blog titles are displayed on the side bar, would this title be potentially too long?

![Screenshot 2024-02-22 at 14 45 42](https://github.com/open-telemetry/opentelemetry.io/assets/4158734/f1b133ed-58b6-4a71-8d91-b686599e7aa0)